Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -895,7 +895,7 @@ final class AttributeTests: ParserTestCase {
         ),
         DiagnosticSpec(
           locationMarker: "2️⃣",
-          message: "unexpected code '))' before macro"
+          message: "unexpected code '))' in class"
```

Comment:
This and similar changes are because the unexpected code are now separate `UnexpectedCodeDeclSyntax` in the statements.